content/posts/0.28-release.md

```diff
@@ -0,0 +1,266 @@
+of Pyodide, [Michael Droettboom](https://droettboom.com/), who was also a core
+developer of Matplotlib.
+
+Another backend, [the `html5_canvas` backend](https://blog.pyodide.org/posts/canvas-renderer-matplotlib-in-pyodide/), was developed by Madhur Tandon as a part of a [Google Summer of Code
+2019 project](https://summerofcode.withgoogle.com/archive/2019/projects/4683094261497856).
+
+However, these backends hadn't been maintained for a long time, and without
+dedicated expertise among our core developers, they became increasingly incompatible
+with newer Matplotlib versions. This made it tough to keep up with new features and
+critical bug fixes.
+
+In this release, we have deprecated these custom backends, and [replaced them](https://github.com/matplotlib/matplotlib/pull/29568) with
+a patched version of
+[the `WebAgg` backend](https://matplotlib.org/stable/api/backend_webagg_api.html),
+one of the official browser-based Matplotlib backends. This new backend provides
+a more stable and feature-rich experience for rendering Matplotlib plots in the browser.
+
+A huge thank you to [Ian Thomas](https://github.com/ianthomas23), a Matplotlib
+core developer and JupyterLite maintainer, who wrote and maintains this new
+backend.
+
+## Other improvements
+
+### Standardized package loading with runtime paths
+
+This release introduces support for runtime paths in Emscripten modules, which
+allows us to correctly locate shared library dependencies.
+
+If a Python binary extension `somebinmod.so` depends on a shared library,
+`libsomedep.so`, this information will be included in the dynamic loader section
+of `somebindmod.so`. The dynamic loader will search for `libsomedep.so` on the
+`LD_LIBRARY_PATH`.
+
+Python wheels vendor their shared libraries, so if `somebinmod.so` is contained
+in `somepackage` then `somepackage.whl` will include a folder called
+`somepackage.libs` with `libsomedep.so` inside. However, `somepackage.libs`
+cannot be added to the `LD_LIBRARY_PATH` because we only want to search that
+directory when opening shared libraries in `somepackage`.
+
+Instead, each dynamic library has its own dependency search path called the
+runtime path. This is information included in the dynamic loader section of a
+shared library indicating where its dependencies should be located. In this
+case, `somebinmod.so` would have an entry saying to look in `somepackage.libs`
+for its dependencies.
+
+Previously, WebAssembly shared libraries files did not support runtime paths, so
+we had to use a custom patch for the Emscripten dynamic loader to apply the rule
+that `somepackage.libs` should be searched when loading libraries from
+`somepackage`. This patch exposed us to extra bugs and prevented us from being
+able to upstream fixes to the Emscripten dynamic loader. It also forced us to
+load dynamic libraries eagerly rather than lazily.
+
+We added runtime path to the WebAssembly specification for the shared library
```

Comment:
```suggestion
We added runtime path to the Emscripten specification for the shared library
```

To be precise, I think it is Emscripten not WebAssembly. There is no spec for dynamic loading in WASM spec.

Reply:
Well it lives here:
https://github.com/WebAssembly/tool-conventions
So I think it's a WebAssembly spec.